views/create-job.njk

```diff
@@ -0,0 +1,233 @@
+                        <div class="flex items-center">
+                            <i data-lucide="alert-circle" class="h-5 w-5 mr-2"></i>
+                            <span>{{ errorMessage }}</span>
+                        </div>
+                    </div>
+                </div>
+            {% endif %}
+
+            <form id="createJobForm" class="bg-white p-8 rounded-lg border border-gray-200 shadow-sm">
+                <!-- Job Title -->
+                <div class="mb-6">
+                    <label for="name" class="block text-gray-700 font-semibold mb-2">
+                        Job Title <span class="text-red-600">*</span>
+                    </label>
+                    <input 
+                        type="text" 
+                        id="name" 
+                        name="name" 
+                        required
+                        class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+                        placeholder="e.g., Senior Software Engineer"
+                    />
+                </div>
+
+                <!-- Location -->
+                <div class="mb-6">
+                    <label for="location" class="block text-gray-700 font-semibold mb-2">
+                        Location <span class="text-red-600">*</span>
+                    </label>
+                    <input 
+                        type="text" 
+                        id="location" 
+                        name="location" 
+                        required
+                        class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+                        placeholder="e.g., Belfast, UK"
+                    />
+                </div>
+
+                <!-- Capability -->
+                <div class="mb-6">
+                    <label for="capability" class="block text-gray-700 font-semibold mb-2">
+                        Capability <span class="text-red-600">*</span>
+                    </label>
+                    <select 
+                        id="capability" 
+                        name="capability" 
+                        required
+                        class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+                    >
+                        <option value="">Select capability...</option>
+                        <option value="Engineering">Engineering</option>
+                        <option value="Platforms">Platforms</option>
+                        <option value="Data">Data</option>
+                        <option value="Artificial Intelligence">Artificial Intelligence</option>
+                        <option value="Cyber Security">Cyber Security</option>
+                        <option value="Workday">Workday</option>
+                        <option value="Experience Design">Experience Design</option>
```

Comment:
it would be great to create those in a loop, and extract the list of available capabilities to some class.
It's generally a good to keep any data / login out of view and just render the data that gets passed to it